docs/install.md

```diff
@@ -0,0 +1,33 @@
+---
+# Discussion Forum
+
+## Install Guide
+---
+ ### Server Installation
+- Download the latest version of WAMP/XAMPP server.
+- Install it through the wizard.
```

Comment:
Again, this is getting too specific towards one OS. I would instead prefer linking to a page that tells how to install a web server with PHP support for all OSes.... But I think this would do for a start.